game_cfg/setup.h

```diff
@@ -0,0 +1,3 @@
+// Map parameters
+#define LVL_WIDTH 80;
```

Comment:
I suspect that these don't belong here. Once we are doing more interesting stuff with levels the level generation code itself will likely determine these values, though there may be some maximum sizes or something here.